### ISSUE TITLE
Staging Review finding: Text clipped on small screens/zoomed in

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#120704]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** govcio-vebt
**Product:** VA Form 22-10297
**Feature:** Application for High Technology Program

## Findings details
**VA.gov Experience Standard - issue:** [User experience is not consistent with the VA.gov design principles.](https://design.va.gov/about/experience-standards/#users-experience-is-consistent-with-the-vagov-design-principles)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On the Review application page, in the Attestation section, the privacy policy text is clipped offscreen at small sccreens (320px) or zoomed in at 400% (at 1280px)

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="305" height="354" alt="clipped privacy policy" src="https://github.com/user-attachments/assets/afbcbe07-cf3b-4f50-b2fd-10a0ec936e66" />

</details>

## Recommended action

Ensure the text wraps appropriately. 

Note: the corresponding CSS that is creating the issues is:

```
.privacy-agreement {
    white-space: nowrap;
}
```

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 1.4.10 Reflow - Level AA](https://www.w3.org/TR/WCAG22/#reflow)
- **Modality:** Browser zoom
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
